apps/Gruntfile.js

```diff
@@ -345,12 +345,18 @@ module.exports = function (grunt) {
           ' -p [ factor-bundle -o ' + allFilesDest.join(' -o ') + ' ] -o ' + outputDir + 'common.js' :
           ' -o ' + allFilesDest[0]
       );
+  var applabAPIExec = 'mkdir -p build/browserified && `npm bin`/browserifyinc' +
+      ' --cachefile ' + outputDir + 'applab-api-cache.json' +
+      ' -t [ babelify --compact=false --sourceMap --sourceMapRelative="$PWD" ]' +
+      (envOptions.dev ? '' : ' -t [ envify --NODE_ENV production ]') +
+      ' -d build/js/applab/api-entry.js -o ' + outputDir + 'applab-api.js';
```

Comment:
It looks like it'd be pretty easy to extract common logic in constructing this build command and the original one above.  Will you do that here?